### ISSUE TITLE
training set images

### ISSUE THREAD

**Issue by raaka1** [tbennun/cudnn-training#14]
train-images-idx3-ubyte.gz:  training set images (9912422 bytes) 
train-labels-idx1-ubyte.gz:  training set labels (28881 bytes) 
t10k-images-idx3-ubyte.gz:   test set images (1648877 bytes) 
t10k-labels-idx1-ubyte.gz:   test set labels (4542 bytes)

do we have to unpack them ?
